packages/orchestrator-ui-components/src/icons/WFOCogFill.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { WFOIconProps } from './WFOIconProps';
+
+export const WFOCogFill: FC<WFOIconProps> = ({
+    width = 24,
+    height = 24,
+    color = '#000000',
+}) => {
+    return (
+        <svg
+            width={width}
+            height={height}
+            viewBox={`0 0 ${width} ${height}`}
```

Comment:
Just as a warning to double check. In other icons you (and myself) did not make the viewBox configurable, just hardcoded 24 24. In my experience I got sometimes not-so-nice icons.

Reply:
This is needed to get the smaller icons from the design. It looks ok at least on my machine.